queue_services/entity-emailer/src/entity_emailer/email_processors/correction_notification.py

```diff
@@ -230,12 +228,10 @@ def process(email_info: dict, token: str) -> Optional[dict]:  # pylint: disable=
         if original_filing_type in ['annualReport', 'changeOfAddress', 'changeOfDirectors']:
             return None
     elif is_special_resolution_correction(
-            filing.filing_json['filing'],
-            business,
-            filing
+            filing.filing_json['filing']
     ):
         prefix = 'CP-SR'
-        name_changed = filing.filing_json['filing']['correction'].get('nameRequest', {})
+        name_changed = 'requestType' in filing.filing_json['filing'].get('correction', {}).get('nameRequest', {})
```

Comment:
Consistent with above, requestType CHG is included on changeOfName for a correction